src/util/forge-config.js

```diff
@@ -3,6 +3,29 @@ import path from 'path';
 import _template from 'lodash.template';
 import readPackageJSON from './read-package-json';
 
+const proxify = (object, keyChain) => {
```

Comment:
@malept FYI this is a generic solution to the whole "I want to pass this config in through environment" issue.  It automagically proxies all requests for config options to the equivalent `process.env.ELECTRON_FORGE_CONFIG_THING` variable.

E.g. `process.env.ELECTRON_FORGE_S3_SECRET_ACCESS_KEY`

Just means we don't need to worry about environment variables and can just hit the config object when we want anything (pretty cool IMO) 😄 Node 6 FTW